Resources/Prototypes/Entities/Objects/Devices/Electronics/door_access.yml

```diff
@@ -278,6 +270,47 @@
   - type: AccessReader
     access: [["SyndicateAgent"]]
 
+# Cross-Department
+- type: entity
+  parent: DoorElectronics
+  id: DoorElectronicsMedicalResearch
+  suffix: Medical/Science, Locked
+  components:
+  - type: AccessReader
+    access: [["Research"], ["Medical"]]
+
+- type: entity
```

Comment:
these should also be in our file structure, under Prototypes/_Impstation/Objects/Devices/Electronics/door_access.yml. as a general rule, if you are adding new entities to an existing file, you should be making a different version of that file in our subfolder instead